index.js

```diff
@@ -1,9 +1,112 @@
 const core = require('@actions/core');
-const github = require('@actions/github');
+const { google } = require('googleapis');
+const fs = require("fs");
 
 async function run() {
-  const input1 = core.getInput("INPUT-1");
-  const input2 = core.getInput("INPUT-2");
+  const path = core.getInput("JACOCO_CSV_PATH");
+  const spreadsheetId = core.getInput("SPREADSHEET_ID");
+  const sheetName = core.getInput("SPREADSHEET_NAME");
+  const apiClientEmail = core.getInput("API_CLIENT_EMAIL");
+  const privateKey = core.getInput("API_PRIVATE_KEY");
+  const codeCoverageData = fs.readFileSync(path).toString().split('\n').map(e => e.trim()).map(e => e.split(',').map(e => e.trim()));
+  var refactoredCodeCoverageData = getRefactorCSVCodeCoverageData(codeCoverageData);
+  const auth = new google.auth.JWT(
+    apiClientEmail,
+    null,
+    privateKey,
+    ['https://www.googleapis.com/auth/spreadsheets']
+  )
+  const googleSheetsInstance = google.sheets({ version: "v4", auth: auth });
+  try {
+    const readData = await googleSheetsInstance.spreadsheets.values.get({
+      auth,
+      spreadsheetId,
+      range: sheetName,
+    })
+    const googleSheetData = generateGoogleSheetData(readData, refactoredCodeCoverageData)
+    updateGoogleSpreadSheet(auth, spreadsheetId, sheetName, googleSheetData, googleSheetsInstance)
+  } catch (e) {
+    console.error(e)
```

Comment:
Shouldn't you terminate the program? This will silently "make it work". You don't want that. I think just not `catch`ing should do the trick?